### ISSUE TITLE
HBaseException make assertion thrown

### ISSUE THREAD

**Issue by manolama** [OpenTSDB/asynchbase#170]
See https://groups.google.com/forum/#!topic/opentsdb/3lw8h6U5D5s

```
org.jboss.netty.util.HashedWheelTimer
WARN: An exception was thrown by TimerTask.
java.lang.AssertionError: Must not be used.
	at org.hbase.async.HBaseException.make(HBaseException.java:62)
	at org.hbase.async.RegionClient$1MultiActionCallback.handleException(RegionClient.java:903)
	at org.hbase.async.RegionClient$1MultiActionCallback.call(RegionClient.java:867)
	at com.stumbleupon.async.Deferred.doCall(Deferred.java:1278)
	at com.stumbleupon.async.Deferred.runCallbacks(Deferred.java:1257)
	at com.stumbleupon.async.Deferred.callback(Deferred.java:1005)
	at org.hbase.async.HBaseRpc.callback(HBaseRpc.java:712)
	at org.hbase.async.HBaseRpc$TimeoutTask.run(HBaseRpc.java:632)
	at org.jboss.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:556)
	at org.jboss.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:632)
	at org.jboss.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:369)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by jasonculverhouse** [comment]
It looks like a few classes derived from `HBaseException `  do not define `org.hbase.async.HBaseException.make(Object, HBaseRpc)` which results in the "Must not be used." message.

    org.hbase.async.ConnectionResetException
    org.hbase.async.RpcTimedOutException
    org.hbase.async.RegionOfflineException

In this case looks like an `org.hbase.async.RpcTimedOutException` 

I would think that you should just make `HBaseException.make` abstract  so that all of the derived classes have to implement a reasonable result?

**Comment by manolama** [comment]
Fixed in 4aa86790830ce7e5e7e99cce72e3282b80ec514e, thanks!
